GOVERNANCE.md

```diff
@@ -0,0 +1,29 @@
+# Policy and Procedure
+
+
+## Maintainers
+
+|Maintainer|Role|
+|---|---|
+|*Dan Harris|CSC Operations|
```

Comment:
We may want to keep the roles aligned with recognized Center categories (groups, teams, etc.). So maybe CSC Operations is ACO (group) or User Operations (team); CSC Applications would be CSSO (group) or User and Applications Support (team). I like the team labels, as it raises visibility of the things we do within the Center and the groups.

Reply:
+1 on teams